core/src/main/resources/defaults/blueprints/7.2.10/cdp-flow-management-small.bp

```diff
@@ -137,8 +137,12 @@
                 "value": "-Xms8g"
               },
               {
-                "name": "add.internal.nifi.registry.client",
-                "value": "true"
+                "name": "nifi.registry.client.name",
+                "value": "Default NiFi Registry Client"
+              },
+              {
+                "name": "nifi.registry.client.url",
+                "value": "https://placeholder:18433"
```

Comment:
Out of curiosity, what is the purpose of this fake URL? Will it be simply ignored or later updated (by CM etc.) to some valid value?

Reply:
@lajosrodek yes, after NiFi will be started, this value will be updated in createRequiredNiFiObjects script (in CSD).

Reply:
Got it, thanks.

Reply:
I think should be part of the csd right ?

Reply:
@doktoric no, it should not be a part of CSD. For private cloud clusters, user should set values for these properties manually.

Reply:
@vbrodetskyi do we always need this property? if yes should we create configprovider for this?

Reply:
@doktoric we definitely need these two properties for 7.2.8 (CFM-2.0.8.0), but in future versions, it could be changed. I don't think we need configprovider, these properties should be enough.